DevAcademy/Components/Toolbar/FilterBar/ToolbarWithFilterByKind.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+
+struct ToolbarWithFilterByKind: View {
+    // MARK: PROPERTIES
+    @Binding var selectedKind: String
+
+    // MARK: - BODY
+    var body: some View {
+            ScrollView(.horizontal, showsIndicators: false) {
+                HStack {
+                    Button {
```

Comment:
Rekl bych, ze ten kind "Vše" by sel pridat primo do toho enumu Kind. Pak by tu nebyl potreba ten extra Button a stacil by tu jenom ten ForEach nad `Kind.allCases`. 
Zaroven by si pak mohla v `selectedKind` pracovat primo s tim enumem Kind a ne jeho `rawValue`. Coz ti taky muze ulehcit praci a eliminovat potencialni chyby.
Nebo je nejaky duvod pro to drzet si ten "Vše" kind mimo enum? 🤔